Demos/Priority/Allocator/PriorityAllocator.py

```diff
@@ -78,7 +91,8 @@ def allocate_path(self, bid: "PriorityPathBid", environment: "Environment", asta
             if environment.is_coordinate_blocked_forever(b, bid.agent.near_radius):
                 return None, None, f"Static blocker at target {b}."
 
-            a_t = find_valid_path_tick(tick, environment, self.bid_tracker, a, bid.agent, tick, environment.dimension.t)
+            a_t = find_valid_path_tick(tick, environment, self.bid_tracker, a, bid.agent, tick - bid.agent.speed,
+                                       environment.dimension.t)
```

Comment:
do we want to always allocate into the past?

Reply:
no, tick - bid.agent.speed is the min time